osf/management/commands/check_crossref_dois.py

```diff
@@ -27,6 +27,24 @@ def pop_slice(lis, n):
     del lis[:n]
     return tem
 
+def create_dois_locally():
+    """
+    This script is to crate pending DOIs locally
+    """
+    preprints_with_pending_dois = Preprint.objects.filter(
+        preprint_doi_created__isnull=True,
+        is_published=True
+    )
+
+    if not preprints_with_pending_dois.exists():
+        return
+
+    preprints = list(preprints_with_pending_dois)
```

Comment:
Django does the magic and evaluates `QuerySet` during a `for` loop. So no need to turn it into a list.

```python
for preprint in preprints_with_pending_doi:
```

With this, we don't even need to do the `None` check since the `for` loop does it for us.